### PR TITLE
Add cli calls to play connectfour

### PR DESCRIPTION
**Pull request by haerdib**
Getter:
`./integritee-cli -p 9994 -P 2094 trusted get-board //Bob --mrenclave=5q8KKAzwH3sfbfurS8VE895mMny3pGkuz4EzueRbw5XR`

Call:
`./integritee-cli -p 9994 -P 2094 trusted play-turn //Bob 2 --mrenclave=5q8KKAzwH3sfbfurS8VE895mMny3pGkuz4EzueRbw5XR --direct`

Not yet tested as no state available currently, but to minimize rebase work, I think we should merge.
(Cli calls are working, just the sgx-runtime state connection is not yet tested)

### CODE REVIEW

File: app-libs/stf/src/cli.rs [hunk 1]
```diff
@@ -330,6 +330,113 @@ pub fn cmd<'a>(
...
+								.required(true)
+								.value_name("SS58")
+								.help("player's AccountId in ss58check format"),
+						)
+						.arg(
+							Arg::with_name("column")
+								.takes_value(true)
+								.required(true)
+								.value_name("u8")
+								.help("play stone in column, must be in the range of [0,7]"),
```
Comment (gaudenzkessler) [inline app-libs/stf/src/cli.rs:350]: there are 7 columns: range [0,6]?

File: app-libs/stf/src/cli.rs [hunk 2]
```diff
@@ -330,6 +330,113 @@ pub fn cmd<'a>(
...
+				})
+				.runner(move |_args: &str, matches: &ArgMatches<'_>| {
+					let arg_player = matches.value_of("player").unwrap();
+					let column = matches
+						.value_of("column")
+						.unwrap()
+						.parse()
+						.expect("amount can be converted to u8");
+
+					if column >= 8 {
```
Comment (gaudenzkessler) [inline app-libs/stf/src/cli.rs:361]: 6?

File: app-libs/stf/src/cli.rs [hunk 3]
```diff
@@ -330,6 +330,113 @@ pub fn cmd<'a>(
...
+						panic!("Game only allows columns smaller then 8");
```
Comment (gaudenzkessler) [inline app-libs/stf/src/cli.rs:362]: 6?

**Review by gaudenzkessler — CHANGES_REQUESTED** [review]
Looks fine only a question on the coulumn numbers

File: app-libs/stf/src/cli.rs (hunk 1 above)
Comment (haerdib) [reply app-libs/stf/src/cli.rs:350]: Hm. I see. Then the check in the connect four is not appropriate: https://github.com/ajuna-network/ajuna-node/blob/update-substrate-5/pallets/ajuna-connectfour/src/lib.rs#L288

Should be [1, 7] then.

Comment (haerdib) [reply app-libs/stf/src/cli.rs]: Updated, thanks for the review!

**Review by gaudenzkessler — APPROVED** [review]
great

**Pull request merged**